addons/bus/websocket.py

```diff
@@ -811,17 +811,28 @@ def open_connection(cls, request):
         :raise: BadRequest if the handshake data is incorrect.
         """
         cls._handle_public_configuration(request)
-        response = cls._get_handshake_response(request.httprequest.headers)
-        response.call_on_close(functools.partial(
-            cls._serve_forever,
-            Websocket(request.httprequest.environ['socket'], request.session),
-            request.db,
-            request.httprequest
-        ))
-        # Force save the session. Session must be persisted to handle
-        # WebSocket authentication.
-        request.session.is_dirty = True
-        return response
+        try:
+            response = cls._get_handshake_response(request.httprequest.headers)
+            socket = request.httprequest.environ['socket']
+            response.call_on_close(functools.partial(
+                cls._serve_forever,
+                Websocket(socket, request.session),
+                request.db,
+                request.httprequest
+            ))
+            # Force save the session. Session must be persisted to handle
+            # WebSocket authentication.
+            request.session.is_dirty = True
+            return response
+        except KeyError as exc:
+            raise RuntimeError(
+                f"Couldn't bind the websocket. Is the connection opened on the evented port ({config['gevent_port']})?"
+            ) from exc
+        except Exception as exc:
+            _logger.error(exc)
+            raise
```

Comment:
```suggestion
```
Just let the other exceptions bubble up, the http stack does the logging already

Reply:
I don't have any other information than the http return code so I think this line should be kept.

![image](https://github.com/odoo/odoo/assets/48757558/0b2b143a-748b-43f7-8e38-61988e940f81)
vs
![image](https://github.com/odoo/odoo/assets/48757558/0a6b1458-4fa7-4568-885f-2e82914d11f8)

Reply:
indeed, `werkzeug.exceptions.HTTPException` and derivated classes are not logged by the http stack, can you replace `except Exception as exc:` for a `except werkzeug.exceptions.HTTPException as exc:` then?

Reply:
done :+1: